backend/app/auth/routers.go

```diff
@@ -1,13 +1,42 @@
 package auth
 
 import (
+	"fmt"
+	"github.com/gin-gonic/contrib/sessions"
 	"github.com/osscameroon/yotas/app"
+	"golang.org/x/oauth2"
+	githuboauth "golang.org/x/oauth2/github"
+	"log"
+	"os"
 )
 
 func AuthRouter() {
 	router := app.GetApiRouter()
 
-	router.POST("/login", LoginHandler)
-	router.GET("/github/login", GithubCallbackHandler)
-	router.POST("/github/login", GithubCallbackHandler)
+	// We get our secret
+	secretKey := os.Getenv("SECRET_KEY_BASE")
+
+	// auth configurations
+	oauthConf := &oauth2.Config{
+		ClientID:     os.Getenv("GITHUB_CLIENT_ID"),
```

Comment:
Instead of hard typing env var here you can use `app.Env.GithubClientId`

Reply:
Good Catch !